src/test/groovy/com/synacy/gradprogram/spock/exercise/ShoppingServiceSpec.groovy

```diff
@@ -0,0 +1,59 @@
+package com.synacy.gradprogram.spock.exercise
+
+import spock.lang.Specification
+
+class ShoppingServiceSpec extends Specification {
+    OrderingService orderingService = Mock(OrderingService)
+    DeliveryService deliveryService = Mock(DeliveryService)
+    OrderRepository orderRepository = Mock(OrderRepository)
+    DeliveryRequestRepository deliveryRequestRepository = Mock(DeliveryRequestRepository)
+
+    ShoppingService shoppingService
+
+    void setup() {
+        shoppingService = new ShoppingService(orderingService, deliveryService,
+                orderRepository, deliveryRequestRepository)
+    }
+
+    def "buyNonSpoilingItemsInCart should create delivery order for given cart and user details"() {
```

Comment:
Missing test for checking the creation of `Order`since that part is also what the method is doing.